README.md

```diff
@@ -100,6 +100,49 @@ You can also use `LocalGitHubOAuthenticator` to map GitHub accounts onto local u
 
 You can use your own Github Enterprise instance by setting the `GITHUB_HOST` environment variable.
 
+### GitHub-specific features
+
+Additional features are turned on by overriding the authenticator's
+`scope` list with particular GitHub scopes:
+
+https://developer.github.com/apps/building-integrations/setting-up-and-registering-oauth-apps/about-scopes-for-oauth-apps/
+
+Some example scopes you might want:
+
+`read:org` grants access to the users' organizations.  This is handy if
+you want to use GitHub organizations in your backend environment as Unix
+groups for collaboration purposes.  Having globally consistent UIDs
+(from the GitHub ID) and GIDs (from the organization IDs) makes access
+permissions on shared storage much easier.
+
+`public_repo` allows read and write of public repositories; if you want
```

Comment:
allows "trusted users" read and write privileges for public repositories. If you want to automatically provision `git` pushes to GitHub, you can accomplish this by passing a token to your Lab or classic Notebook instance.